src/Repository/MajorVersionRepository.php

```diff
@@ -88,40 +95,49 @@ public function findAllActiveElts()
         return $qb->getQuery()->execute();
     }
 
-    public function findAllPreparedForJson()
+    /**
+     * @return mixed[]
+     */
+    public function findAllPreparedForJson(): array
     {
         $data = $this->findCommunityVersionsGroupedByMajor();
         $data = array_merge($data, $this->findStableReleases());
-        $data = array_merge($data, $this->findLtsReleases());
-        return $data;
+        return array_merge($data, $this->findLtsReleases());
     }
 
+    /**
+     * @return array<string, MajorVersion>
+     */
     public function findAllGroupedByMajor(): array
     {
-        $all = $this->findAll();
+        $majorVersions = $this->findAll();
         $data = [];
-        foreach ($all as $version) {
-            $data[$this->formatVersion($version->getVersion())] = $version;
+        foreach ($majorVersions as $majorVersion) {
```

Comment:
I'd probably just stick `$version` here for the individual variable.  Mostly personal preference, I think, but `$majorVersion` just seems needlessly verbose.  s/$all/$majorVersions/ is fine, though.

Reply:
I've replaced $all with $majorVersions and that's why rector changed the second part. I think I can change $all to $versions and $version will be unchanged. With $all rector changed it to $allSingle which is evil

Reply:
done